### PR TITLE
Encryption ADR

### PR DESCRIPTION
**Pull request by Hinton**

### CODE REVIEW

**Review by MGibson1 — COMMENTED** [review]
This seems ready to go, just one question about your ciphers callout

File: docs/architecture/adr/0016-decryption-in-views.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  follows the single responsibility principle.
+- Domain models are no longer tightly coupled to views.
+- We can now have multiple views per domain.
+
+### Negative Consequences
+
+- Ciphers still needs a method on the `<Domain>Service` since it currently calculates the password
+  history on save. This could be argued is actual a domain concern, and decoupling the history from
+  encryption is a good thing. It could still be confusing since it's possible to bypass the
+  `CipherService` which would cause the history to not be updated.
```
[the remark this answers is not in the document]
Comment (Hinton) [reply docs/architecture/adr/0016-decryption-in-views.md:79]: https://github.com/bitwarden/clients/pull/4473/files#diff-dfa6127fe9628b5760071f5679b3c20af10e27ec3708eda842d58ee2efbe6fb0R90

[the remark this answers is not in the document]
Comment (MGibson1) [reply docs/architecture/adr/0016-decryption-in-views.md:79]: I follow, I just think that's an issue with the implementation of histories, not necessarily with encrypt/decrypt.

I'm not convinced it's worth a callout here, but it's not really hurting anything

**Review by MGibson1 — APPROVED** [review]
Approving because I don't feel strongly about the history thing, lmk if you decide to rework the negative consequences to be a little more general.

**Pull request merged**